### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by sobolevn** [3YOURMIND/django-migration-linter#42]
Hi, I am using this project template for my project: https://github.com/wemake-services/wemake-django-template

I have installed `django-migration-linter` and used it like so: `django-migration-linter .`

That's how it failed:

```python
» django-migration-linter .
Traceback (most recent call last):
  File "/Users/sobolev/Documents/PyCharmProjects/rostelecom_svetochka/.venv/bin/django-migration-linter", line 11, in <module>
    sys.exit(_main())
  File "/Users/sobolev/Documents/PyCharmProjects/rostelecom_svetochka/.venv/lib/python3.6/site-packages/django_migration_linter/migration_linter.py", line 339, in _main
    no_cache=args.no_cache,
  File "/Users/sobolev/Documents/PyCharmProjects/rostelecom_svetochka/.venv/lib/python3.6/site-packages/django_migration_linter/migration_linter.py", line 70, in __init__
    self.old_cache = Cache(self.django_path, self.cache_path)
  File "/Users/sobolev/Documents/PyCharmProjects/rostelecom_svetochka/.venv/lib/python3.6/site-packages/django_migration_linter/cache.py", line 26, in __init__
    project_name = split_path(django_folder)[-2]
IndexError: list index out of range
```

I have also tried to use `django-migration-linter server`, but is said that `server` is not a `django` app.
As a result I was not able to run this linter for my setup.

Do you have any ideas how it can be solved? Thanks!

**Comment by sobolevn** [comment]
Related https://github.com/wemake-services/wemake-django-template/issues/634

**Comment by David-Wobrock** [comment]
Hi @sobolevn , thanks a lot for the submitted issues.

This bug should be fixed on the latest state of `master` - I am just waiting for a confirmation and I'll do a release for it :)

**Comment by David-Wobrock** [comment]
I released a new version `0.1.2` - it should work now :)

Don't hesitate to re-open the issue if you still encounter a problem
